docs/core/whats-new/dotnet-core-2-2.md

```diff
@@ -0,0 +1,100 @@
+            Console.WriteLine($"\tName = \"{eventData.PayloadNames[i]}\" Value = \"{payloadString}\"");
+        }
+        Console.WriteLine("\n");
+    }
+}
+```
+
+In addition, .NET Core 2.2 adds the following two properties to the <xref:System.Diagnostics.Tracing.EventWrittenEventArgs> class to provide additional information about ETW events:
+
+- <xref:System.Diagnostics.Tracing.EventWrittenEventArgs.OSThreadId?displayProperty=nameWithType>
+
+- <xref:System.Diagnostics.Tracing.EventWrittenEventArgs.TimeStamp?displayProperty=nameWithType>
+
+## Data
+
+**AAD authentication to Azure SQL databases with the SqlConnection.AccessToken property**
+
+Starting with .NET Core 2.2, an access token issued by Azure Active Directory can be used to authenticate to an Azure SQL database. To support access tokens, the <xref:System.Data.SqlClient.SqlConnection.AccessToken> property has been added to the <xref:System.Data.SqlClient.SqlConnection> class. To take advantage of AAD authentication, download version 4.6 of the System.Data.SqlClient NuGet package.
+
+## JIT compiler improvements
+
+**Tiered compilation is enabled by default**
+
+In .NET Core 2.1, the JIT compiler implemented a new compiler technology, *tiered compilation*. The goal of tiered compilation is improved performance. One of the important tasks performed by the JIT compiler is optimizing code execution. For little-used code paths, however, the compiler may spend more time optimizing code than the runtime spends executing unoptimized code. Tiered compilation introduces two stages in JIT compilation:
+
+- A **first tier**, which generates code as quickly as possible.
+
+- A **second tier**, which generates optimized code for those methods that are executed frequently. The second tier of compilation is performed in parallel for enhanced performance.
+
+For information on the performance improvement that can result from tiered compilation, see [Announcing .NET Core 2.2 Preview 2](https://blogs.msdn.microsoft.com/dotnet/2018/09/12/announcing-net-core-2-2-preview-2/). 
+
+In .NET Core 2.1, tiered compilation was disabled by default; you had to explicitly opt in to take advantage of it. In .NET Core 2.2, on the other hand, tiered compilation is enabled by default. You can still disable tiered compilation in either of two ways.
+
+- To disable tiered compilation in all projects that use the .NET Core 2.2 SDK, set the following environment variable:
+
+  ```console
+  COMPlus_TieredCompilation="0"
+  ```
+
+- To disable tiered compilation on a per-project basis, add the `<TieredCompilation>` property to the `<PropertyGroup>` section of the MSBuild project file, as the following example shows:
+
+   ```xml
+   <PropertyGroup>
+      <!-- other property definitions -->
+
+      <TieredCompilation>false</TieredCompilation>
+   </PropertyGroup>
+   ```
+
+## See also
+
+* [What's new in .NET Core](index.md)  
+* [New features in EF Core 2.2](/ef/core/what-is-new/ef-core-2.2)  
```

Comment:
should there be a pointer to what's new in ASP.NET Core as well?

Reply:
Yes @Andrew-MSFT. @scottaddie @Rick-Anderson do you know the final link for asp.net 2.2 what's new already?

Reply:
I don't believe this doc exists, but it should. @Rick-Anderson Is there an issue tracking the creation of this doc?

Reply:
`[What's new in ASP.NET Core 2.2](/aspnet/core/release-notes/aspnetcore-2.2)`
[Internal review link](https://review.docs.microsoft.com/en-us/aspnet/core/release-notes/aspnetcore-2.2?view=aspnetcore-1.0&branch=pr-en-us-9786)

Reply:
cc @tdykstra